scripts/github-actions-time.py

```diff
@@ -0,0 +1,170 @@
+import os
```

Comment:
It would be good to add a description of what this script does, maybe in the README?